RELEASE.md

```diff
@@ -0,0 +1,4 @@
+Release type: patch
+
+Fix a bug ''a custom resolver with permission_classes doesn't accept @sync_to_async decorator"
```

Comment:
```suggestion
Fix a bug "a custom resolver with permission_classes doesn't accept @sync_to_async decorator"
```

also this seems weirdly written... maybe "fix a bug where doing X would raise an issue Y" or something? not really sure as I'm usually very bad at this =P

Reply:
@bellini666 I modified the release note again. "Fix a bug "StrawberryResolver.is_async returns False for a function decorated by @sync_to_async" I believe this phrase, although not the best, clearly expresses the revisions.